libraries/AP_HAL_ChibiOS/hwdef/PyroF4OSD/hwdef.dat

```diff
@@ -0,0 +1,153 @@
+# hw definition file for PyroDrone F4 OSD
+# WORK IN PROGRESS (NOT TESTED)
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 125
```

Comment:
Need a different board id. There is a list somewhere of the current ones. Pick the next new one.

Reply:
Thanks, wasn't sure what I was supposed to do there. I'll see if I can find the list and change it.